src/main/webapp/jsp/instructorFeedbackResultsByGiverQuestionRecipient.jsp

```diff
@@ -6,7 +6,7 @@
   <script type="text/javascript" src="/js/instructorFeedbackResultsGQR.js"></script>
 </c:set>
 
-<results:resultsPage pageTitle="TEAMMATES - Feedback Session Results" bodyTitle="Session Results" jsIncludes="${jsIncludes}">
+<results:resultsPage pageTitle="Feedback Session Results" bodyTitle="Session Results" jsIncludes="${jsIncludes}">
```

Comment:
Just highlighting that if we drop the `pageTitle` attribute, the title would simply be `Session Results [Instructor]`.